core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature

```diff
@@ -0,0 +1,1155 @@
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 4000                    |
+      | lp2   | 0.9387755102040816 | 50000                   |
+    
+    # Trigger entry into next market period
+    And the network moves ahead "1" blocks:
+
+    # Confirm equity-like-shares are unchanged by the network moving forwards (as virtual-stakes scaled by same factor, r)
+    Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 4000                    |
+      | lp2   | 0.9387755102040816 | 50000                   |
+
+    # -------------------------------------------------------------------------------------------------------------------
+    # -------------------------------------------------------------------------------------------------------------------
+
+
+    # 2nd period: 1 LPs increase commitment, positive growth:
+    # -------------------------------------------------------------------------------------------------------------------
+
+    # Check equity-like-shares before liquidity amendment
+    Given the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share  | average entry valuation |
+      | lp1   | 0.0612244897959184 | 4000                    |
+      | lp2   | 0.9387755102040816 | 50000                   |
+
+    When the parties submit the following liquidity provision:
+      | id  | party | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type   |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | buy  | BID              | 1          | 2      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | buy  | MID              | 3          | 1      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | sell | ASK              | 1          | 2      | amendment |
+      | lp1 | lp1   | ETH/MAR22 | 4000              | 0.001 | sell | MID              | 3          | 1      | amendment |
+
+    # Confirm equity-like-shares updated immediately after liquidity amendment
+    Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share    | average entry valuation |
+      | lp1   | 0.0794314813996635   | 15985.90375384367775    |
+      | lp2   | 0.9205685186003365   | 50000                   |
+
+    # ERROR:
+    # - "equity-like-share" values calculated incorrectly.
+    # - "average-entry-valuation" values calculated incorrectly.
+
+    # EXPECTED:
+    # Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+    #  | party | equity like share | average entry valuation |
+    #  | lp1   | 0.079809          | 15628.748               |
+    #  | lp2   | 0.920191          | 50000.000               |
```

Comment:
please tidy up the comments on the expected value on average entry valuation as they are different from the value in line 236

Reply:
Hey, the values are different from in line 236 as there is an error still.

Values in the "EXPECTED:" section have been updated to match values from the [sheet](https://docs.google.com/spreadsheets/d/1ywOsc9QivCzyKE0TNgflD9EQjZ9cXGiSTiW3efEy22w/edit#gid=1778207922) after the spec change.